code/modules/projectiles/guns/energy/laser.dm

```diff
@@ -67,7 +67,7 @@
 	flags =  CONDUCT
 	slot_flags = SLOT_BACK
 	origin_tech = "combat=4;materials=3;powerstorage=3"
-	ammo_type = list(/obj/item/ammo_casing/energy/laser/heavy)
+	ammo_type = list(/obj/item/ammo_casing/energy/laser/accelerator)

```

Comment:
You did not test this? At least provide a damage graph so we can tell how big of a nerf/buff this is. I am not interested in your "infinite damage" memes, I want to know what this thing will realistically do in most situations.